lms/templates/ux/reference/instructor_dashboard/cohorts.html

```diff
@@ -0,0 +1,188 @@
+                                <option class="option-unavailable">Another deleted content group</option>
+                              </optgroup>
+                            </select>
+
+                            <div class="msg error inline">
+                              <p class="copy-error"><i class="icon icon-warning-sign"></i> The selected content group has been deleted, you may wish to reassign this cohort group.</p>
+                            </div>
+                          </div>
+                        </div>
+                    </div>
+                    <!-- // -->
+
+                    <!-- When no options are available. -->
+                    <h4 class="cohort-section-header divider-lv2">Is this cohort group associated with course-based content groups?</h4>
+                    <div class="form-fields">
+                      <div class="cohort-management-details-assn-course field field-radio">
+                        <label for="cohort-assn-course-no"><input type="radio" name="cohort-assn-course-2" value="no" /> No</label>
+                        <div class="input-group has-other">
+                          <label for="cohort-assn-course-yes"><input type="radio" name="cohort-assn-course-2" value="yes" checked="checked" /> Yes</label>
+                          <div class="input-group-other is-visible">
+                            <div class="msg error inline">
+                              <p class="copy-error"><i class="icon icon-warning-sign"></i> You haven't configured any content groups yet. You need to create a content group before you can create assignments. <a href="#">Create a content group</a></p>
+                            </div>
+                          </div>
+                        </div>
+                    </div>
+                    <!-- // -->
+
+                    <div class="form-actions">
+                        <button type="submit" class="form-submit action-primary">Save</button>
+                        <button type="button" class="button-link action-tertiary">Cancel</button>
+                    </div>
+                </div>
+
+                <div class="wrapper-cohort-supplemental">
+                  <hr class="divider divider-lv1">
+                  <a class="toggle-cohort-management-secondary" href="#cohort-management-file-upload">Assign students to cohort groups by uploading a CSV file</a>
+                  <div class="cohort-management-file-upload csv-upload is-hidden" id="cohort-management-file-upload"></div>
+
+                  <div class="cohort-management-supplemental">
+                      <p class=""><i class="icon icon-info-sign"></i> To review student cohort group assignments or see the results of uploading a CSV file, download course profile information or cohort results on <a href="" class="link-cross-reference" data-section="data_download"> the Data Download page</a>.</p>
```

Comment:
This is an example of when text that represents the icon is not adjacent. Here, I recommend adding the ```aria-hidden="true"``` to the icon and then adding screen reader only text "Note:"